README.md

```diff
@@ -146,7 +146,7 @@ An initial agenda will be posted to the [shared Google docs][shared-meeting-note
 
 Meeting recordings are uploaded to official etcd [YouTube channel].
 
-Get calendar invitation by joining [etcd-dev](https://groups.google.com/forum/?hl=en#!forum/etcd-dev) mailing group.
+Get calendar invitation by joining [etcd-dev](https://groups.google.com/g/etcd-dev) mailing group.
 
 Join Hangouts Meet: [meet.google.com/umg-nrxn-qvs](https://meet.google.com/umg-nrxn-qvs)
```

Comment:
We recently migrated to CNCF Zoom for meetings, would you mind please also updating this link to the zoom link specified on the website?  Refer: https://github.com/etcd-io/website/pull/714

Reply:
sure, @jmhbnz

Reply:
done